corehq/apps/accounting/forms.py

```diff
@@ -1033,10 +1036,12 @@ def create_plan(self):
         description = self.cleaned_data['description']
         edition = self.cleaned_data['edition']
         visibility = self.cleaned_data['visibility']
+        is_customer_software_plan = self.cleaned_data['is_customer_software_plan']
         plan = SoftwarePlan(name=name,
                             description=description,
                             edition=edition,
-                            visibility=visibility)
+                            visibility=visibility,
+                            is_customer_software_plan=is_customer_software_plan)
```

Comment:
nit: I find it useful to add these like 
```
    visibility=visibility,
    is_customer_software_plan=is_customer_software_plan,
)
```
Unless it causes an issue,
So that next time someone adds something to that list of args, the diff is a bit more specific.